Apps/WebClient/src/ClientApp/src/views/faq.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <div class="m-3 px-4">
+        <b-row>
+            <b-col>
+                <b-row class="my-3 fluid">
+                    <b-col>
+                        <div id="pageTitle">
+                            <h1 id="subject">Frequently Asked Questions</h1>
+                            <hr />
+                        </div>
+                    </b-col>
+                </b-row>
+            </b-col>
```

Comment:
two sets of <b-row> and <b-col> not necessary

Reply:
The two sets of rows and cols are still here

Reply:
Oh I see, I thought it was on the FAQ list. Thanks @FuriousLlama